nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/service/AuthorizationService.java

```diff
@@ -461,6 +462,16 @@ public AccessPolicy deleteAccessPolicy(String identifier) {
                         .map(AuthorizationService::resourceToDTO)
                         .collect(Collectors.toList());
 
+        // add any buckets that have been made public for read requests
```

Comment:
I could be missing something, but given that this service is injected with the authorizer (which should be the new Framework authorizer), shouldn't the existing code that looks up authorized resources find the publicly accessible ones? I guess it does not because you added the below and it's not generating duplicates so I guess my question is really just why doesn't the existing stream filter work, for my own understanding?

Reply:
This is a good point. I need to investigate more, but I remember not getting the public buckets until I updated this, although it could be something with the ordering of how I developed things, maybe I changed this part before fully implementing the framework authorizer logic. I'll test this out more and report back.

Reply:
Went back and looked at this and you were right, this change was not necessary since it ends up checking each bucket resource against the framework authorizer which then authorizes the public buckets. 

I pushed another commit that removes those changes and addresses the isLoginSupported change. Thanks!

Reply:
So it looks like to me that now the `getBucketsAllowingPublicRead()` methods that were added to `RegistryService` and `MetadataService` is no longer needed, as this was the only place it was accessed (aside from test code). Do you want to leave it or remove it?